src/models/SpecificationUrl.ts

```diff
@@ -0,0 +1,30 @@
+import fetch from 'node-fetch';
+
+export default class SpecificationURL {
```

Comment:
Isn't SpecificationURL class just checking a URL? I mean, it has nothing to do with the spec but just fetching from a URL and validating a URL. Maybe we can just add these functions to a "helper" library? Also, I noticed you're not using this class anywhere in the code. Is it because it's a Draft PR?

Reply:
will not have the `SpecificationURL` class just checking different approaches